adapters/rubicon/rubicon.go

```diff
@@ -238,35 +233,18 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
 	numRequests := len(request.Imp)
 	errs := make([]error, 0, len(request.Imp))
-	var err error
 	requestData := make([]*adapters.RequestData, 0, numRequests)
 	headers := http.Header{}
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
 	headers.Add("User-Agent", "prebid-server/1.0")
 
-	requestImpCopy := request.Imp
+	impsToExtMap := prepareImpsToExtMap(createImpsToExtMap(request.Imp, errs))
```

Comment:
I do not believe passing the `errs` slice in this way will update the slice in the current context. Like the append function returns a slice to make sure it is properly updated, you will have to do the same here to capture any errors generated within this function.